frontend/public/components/utils/kebab.tsx

```diff
@@ -486,53 +486,40 @@ export const ResourceKebab = connectToModel((props: ResourceKebabProps) => {
   );
 });
 
-class KebabWithTranslation extends React.Component<
-  KebabProps & WithTranslation,
-  { active: boolean }
-> {
-  static factory: KebabFactory = kebabFactory;
-
-  // public static columnClass: string = 'pf-c-table__action';
-  public static columnClass: string = 'dropdown-kebab-pf pf-c-table__action';
-
-  private dropdownElement = React.createRef<HTMLButtonElement>();
-
-  private divElement = React.createRef<HTMLDivElement>();
-
-  constructor(props) {
-    super(props);
-    this.state = {
-      active: false,
-    };
-  }
+const KebabWithTranslation: React.FC<KebabProps> & {
+  factory?: KebabFactory;
+  columnClass?: string;
+} = (props) => {
```

Comment:
Extract the type for readability and export this FC directly because we no longer need the HOC wrapper:

```suggestion
// After L679 where KebabFactory type is defined
type KebabStaticProperties = { 
  columnClass: string 
  factory: KebabFactory;
  getExtensionsKebabActionsForKind: (kind: K8sModel) => KebabAction[];
}
type KebabComponent = React.FC<KebabProps> & KebabStaticProperties
export const Kebab: KebabComponent = (props) => {
```

Reply:
Also remove `t`, `columnClass`, and `factory` from KebabProps. If we remove the `withTranslation` HOC, the `t` prop is no longer passed in. The other two are not React component props, they are static properties defined on the Kebab object itself.